pkg/overlay/cache.go

```diff
@@ -183,6 +169,24 @@ func (cache *Cache) Get(ctx context.Context, nodeID storj.NodeID) (_ *NodeDossie
 	return cache.db.Get(ctx, nodeID)
 }
 
+// IsNew checks if a node is 'new' based on the collected statistics.
+func (cache *Cache) IsNew(node *NodeDossier) bool {
+	return node.Reputation.AuditCount > cache.preferences.AuditCount
```

Comment:
wouldn't it only be new if the audit count was less than the preferred audit count? Not greater than?

Reply:
nice catch!  fixed.

Reply:
(obviously IsNew() is not yet used in the codebase)